training_methods/training_method.py

```diff
@@ -25,6 +25,39 @@ def display_name(self) -> str:
         """
         pass
 
+    @abc.abstractmethod
+    def dataset_time_series_identifier_column(
```

Comment:
These parameters change depending on the training method, so must be defined by each training method.